flaml/tune/tune.py

```diff
@@ -78,6 +78,9 @@ def compute_with_config(config):
         _metric: Optional default anonymous metric for ``tune.report(value)``.
             (For compatibility with ray.tune.report)
         **kwargs: Any key value pair to be reported.
+
+    Returns:
+        A bool indicating whether the trial is finished or not.

```

Comment:
This seems to indicate "True" means the trial is finished.
We can define it that way, but then need to change other places.

Reply:
Another idea: What if we raise StopIteration instead of returning a bool?